src/main/resources/io/jenkins/plugins/synopsys/security/scan/extension/freestyle/SecurityScanFreestyle/help-coverity_project_name.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Project name in Coverity
+    Coverity Project Name optional for multi-branch pipeline, for freestyle and pipeline it is mandatory
```

Comment:
This help text seems less understandable. 
Need to check if we can put something like this  - "The project name in Coverity is optional for multi-branch pipeline, but it is mandatory for freestyle and pipeline"

> Other help text needs to be updated like this if we update this.